styles/Global.css

```diff
@@ -88,6 +141,7 @@ a:hover {
 .nav--menu__on_screen {
 	top: 0;
 	left: 0;
+	position: fixed;
```

Comment:
Not needed. The position type doesn't, or shouldn't change with on or off screen.